tests/unit/src/Internal/Payment/State/ProcessedStateTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Class ProcessedStateTest
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Tests\Internal\Payment\State;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use WC_Helper_Intention;
+use WCPay\Internal\Payment\PaymentContext;
+use WCPay\Internal\Payment\State\CompletedState;
+use WCPay\Internal\Payment\State\ProcessedState;
+use WCPay\Internal\Payment\State\StateFactory;
+use WCPay\Internal\Service\OrderService;
+use WCPAY_UnitTestCase;
+
+/**
+ * Tests for the processed payment state.
+ */
+class ProcessedStateTest extends WCPAY_UnitTestCase {
+	/**
+	 * State under test.
+	 *
+	 * @var ProcessedState
+	 */
+	private $sut;
+
+	/**
+	 * @var StateFactory|MockObject
+	 */
+	private $mock_state_factory;
+
+	/**
+	 * @var OrderService|MockObject
+	 */
+	private $mock_order_service;
+
+	/**
+	 * Set up the test.
+	 */
+	protected function setUp(): void {
+		parent::setUp();
+
+		$this->mock_state_factory = $this->createMock( StateFactory::class );
+		$this->mock_order_service = $this->createMock( OrderService::class );
+		$this->mock_context       = $this->createMock( PaymentContext::class );
```

Comment:
The `$mock_context` property is not defined, let's define it.

Reply:
Addressed here: https://github.com/Automattic/woocommerce-payments/pull/7471/commits/97345dbc66b1cc740585d7850147f228ccdad2eb